src/main/kotlin/app/revanced/patches/youtube/layout/comments/bytecode/patch/CommentsPatch.kt

```diff
@@ -63,6 +67,94 @@ class CommentsPatch : BytecodePatch(
             """
         )
 
+    // hide fullscreen live chat button
+        val liveChatFullscreenButtonMethod = LiveChatFullscreenButtonFingerprint.result!!.mutableMethod
+        val liveChatButtonIndex = liveChatFullscreenButtonMethod.implementation?.instructions?.indexOfFirst {
+            it.opcode.ordinal == Opcode.CONST.ordinal &&
+            (it as WideLiteralInstruction).wideLiteral == CommentsResourcePatch.liveChatButtonId
+        }!!
+
+        liveChatFullscreenButtonMethod.addInstruction(
+            liveChatButtonIndex + 1, """
+                sput v${(liveChatFullscreenButtonMethod.instruction(liveChatButtonIndex) as OneRegisterInstruction).registerA}, Lapp/revanced/integrations/patches/HideLiveChatFullScreenButtonPatch;->fullScreenLiveChatButtonId:I
```

Comment:
There are multiple very long and convoluted lines of duplicate code containing the full descriptor to methods from the integrations. Create variables for the descriptors to deduplicate.